README.md

```diff
@@ -51,7 +46,7 @@ jobs:
       - name: Run chart-releaser
         uses: helm/chart-releaser-action@v1.0.0-rc.1
         env:
-          CR_TOKEN: "${{ secrets.CR_TOKEN }}"
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
If I understand correctly, we don't need to users to add the `CR_TOKEN` option at all since GitHub injects `GITHUB_TOKEN` secret also as an environment variable into workflow steps. This LGTM except we should also remove these lines from the readme example workflow too.

```diff
-        env:
-          CR_TOKEN: "${{ secrets.CR_TOKEN }}"
```

Reply:
We do need this. `chart-releaser` needs the token to create the release.

Reply:
Ah yes, I see what you mean. For `cr upload`.

In that case, I think we should decide which way we want to go with this:
- Allow users to configure the `CR_TOKEN` (in which case we should just add `CR_TOKEN="${CR_TOKEN:-$GITHUB_TOKEN}"` to the top of `cr.sh`)
- Or, always use `GITHUB_TOKEN` (users can't configure), in which case just pass `cr upload --token $GITHUB_TOKEN` when we call it

Reply:
@scottrigby https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token
you still need to pass it since it is in the secrets context, and not as an env var